### DISCUSSION TITLE
UFDR Indexing (Cellebrite)

### DISCUSSION THREAD

**Discussion by benevenuti15** [sepinf-inc/IPED#1398] [General]
I have a UFED touch address, without UFDR format, containing several ".z*" files.Because it is very large, I am not able to open it using Cellebrite Reader, I would like to know if IPED performs the indexing of this type of file, already performing a decompression and receive from the content. If the answer is yes, is there any parameter that must be entered, no indexing time?

**Comment by lfcnassif** [comment]
Just *.ufdr files, single or with multiple segments, are supported currently.
